lunamark/reader/markdown.lua

```diff
@@ -708,16 +784,23 @@ function M.new(writer, options)
                       return larsers.inlines_nbsp
                     end)
 
+  local parse_attributes
+    = create_parser("parse_attributes",
+                    function()
+                      -- N.B. This one uses a global parser
+                      return parsers.attributes
+                    end)
+
   local parse_markdown
   
   ------------------------------------------------------------------------------
   -- Basic parsers (local)
   ------------------------------------------------------------------------------
 
   if options.smart then
-    larsers.specialchar       = S("*_`&[]<!\\'\"-.@^")
+    larsers.specialchar       = S("*_~^`&[]<!\\'\"-.@^")
```

Comment:
`^` is listed twice.

P.S.: Food for thought: should even disabled syntax extensions contribute to `larsers.specialchar`? This seems to make lunamark both backward- and forward-incompatible.

Reply:
```suggestion
    larsers.specialchar       = S("*_~`&[]<!\\'\"-.@^")
```

Reply:
Done